ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -104,14 +84,18 @@ export default class ClientsCountsRoute extends Route {
   }
 
   async model(params: ClientsCountsRouteParams) {
-    const { config, versionHistory } = this.modelFor('vault.cluster.clients') as ClientsRouteModel;
-    // only enterprise versions will have a relevant billing start date, pass null so community users must select initial start time
-    const startTime = this.version.isEnterprise ? getUnixTime(config.billingStartTimestamp) : null;
+    const { config, versionHistory } = this.modelFor('vault.cluster.clients') as ModelFrom<ClientsRoute>;
+    // only enterprise versions will have a relevant billing start date, if null users must select initial start time
+    let startTime = null;
+    if (this.version.isEnterprise && this._hasConfig(config)) {
+      startTime = getUnixTime(config.billingStartTimestamp);
+    }
+
     const startTimestamp = Number(params.start_time) || startTime;
     const endTimestamp = Number(params.end_time) || getUnixTime(timestamp.now());
-    const [activity, activityError] = await this.getActivity(startTimestamp, endTimestamp);
+    const { activity, activityError } = await this.getActivity(startTimestamp, endTimestamp);
```

Comment:
future thought: `getActivity` might be something we can centralize as a util -- I think I saw this change elsewhere already?

Reply:
Yeah, that's a great idea! The other place we query for activity is the dashboard, but I don't think there's a util just yet.

I had planned to make a cleanup ticket because I noticed that the dashboard client count logic is out of date - it still queries the license instead of the counters/config like the client count routes were updated to. I was considering extracting the getActivity request + fallbacks if there is no billing start date so the logic is the same in both places.